src/main/kotlin/tech/thatgravyboat/skyblockpv/screens/tabs/inventory/SacksScreen.kt

```diff
@@ -0,0 +1,38 @@
+package tech.thatgravyboat.skyblockpv.screens.tabs.inventory
+
+import com.mojang.authlib.GameProfile
+import net.minecraft.world.item.ItemStack
+import tech.thatgravyboat.skyblockpv.api.ItemAPI
+import tech.thatgravyboat.skyblockpv.api.data.SkyBlockProfile
+import tech.thatgravyboat.skyblockpv.data.repo.SackCodecs
+import tech.thatgravyboat.skyblockpv.utils.Utils.rightPad
+import tech.thatgravyboat.skyblockpv.utils.Utils.shorten
+import tech.thatgravyboat.skyblockpv.utils.displays.Display
+import tech.thatgravyboat.skyblockpv.utils.displays.Displays
+import tech.thatgravyboat.skyblockpv.utils.displays.asTable
+
+class SacksScreen(gameProfile: GameProfile, profile: SkyBlockProfile? = null) : BasePagedInventoryScreen(gameProfile, profile) {
+    private val sackItems get() = profile?.inventory?.sacks ?: emptyMap()
+    private val sackDisplays: Map<ItemStack, Display>
```

Comment:
make this a `List<Display>` after applying the other suggestion

Reply:
also maybe add some sort of protection in case there is not a single sack present